app/models/payment_intent.rb

```diff
@@ -47,7 +47,7 @@ class PaymentIntent < ApplicationRecord
   validates_uniqueness_of :square_payment_id, allow_nil: true
   has_many :items
   belongs_to :purchaser, class_name: 'Contact'
-  belongs_to :recipient, class_name: 'Contact'
+  belongs_to :recipient, class_name: 'Contact', optional: true
```

Comment:
If you remove this constraint, do you mind adding an assert in the GiftCardCreator that says that in order to create a gift card we need a recipient?